ui/scripts/gen-story-md.js

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env node
 /* eslint-disable */
+// run this script via yarn in the ui directory:
+// yarn gen-story-md some-component
+//
+// or if the story is for a component in an in-repo-addon or an engine:
+// yarn gen-story-md some-component name-of-engine
```

Comment:
i think this note should be added to the Storybook instructions in the README as well!